### PR TITLE
docs: Remove additionalCategories from required fields

### PR DESCRIPTION
**Pull request by dlipovetsky**
**What problem does this PR solve?**:
The other fields are required; we should point out that additionalCategories is optional.

Thanks @deepakm-ntnx for spotting this!

**Which issue(s) this PR fixes**:
Fixes #

**How Has This Been Tested?**:

**Special notes for your reviewer**:

### CODE REVIEW

**Review by jimmidyson — APPROVED** [review]

**Comment by dkoshkin** [comment]
Oops I made some changes along with my changes https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/c714a76830f513abc7a01680ddb9dacbadbf7392/docs/content/customization/nutanix/machine-details.md

How does this PR not have a merge conflict 🤔

**Comment by dkoshkin** [comment]
FYI @dlipovetsky you should have write permissions to this repo. It just makes it easier for the rest of us to checkout the branch and test locally 😄

File: docs/content/customization/nutanix/machine-details.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ spec:
           controlPlane:
             nutanix:
               machineDetails:
+                additionalCategories: # This field is optional.
```
Comment (deepakm-ntnx) [inline docs/content/customization/nutanix/machine-details.md:24]: @dlipovetsky pls refer to https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/commit/f17b1609a1faf1ee90b7cf2b1464c0b724a16239#diff-7be729f26dc4d92f63a4ca892c88ed87a9c4a4b018219c866e41d514c633a199R9. i would prefer that approach of required and then optional1..n as separate sections.

Comment (dkoshkin) [reply docs/content/customization/nutanix/machine-details.md:24]: Ahh I forgot to remove `additionalCategories` from the "required" section 🤦

**Comment by jimmidyson** [comment]
> FYI @dlipovetsky you should have write permissions to this repo. It just makes it easier for the rest of us to checkout the branch and test locally 😄

It's easy enough even from forks to just use `gh pr checkout <num>`

File: docs/content/customization/nutanix/machine-details.md (hunk 1 above)
Comment (dlipovetsky) [reply docs/content/customization/nutanix/machine-details.md]: I'll do that in this PR :smile:

**Review by dkoshkin — APPROVED** [review]

**Review by jimmidyson — APPROVED** [review]

**Pull request merged**